src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -430,6 +446,9 @@ const PlrDaoStakingTransactionBlock = ({
       : totalKeyBasedPLRTokens > 0
       ? 'Key Based'
       : 'Smart Wallet';
+  const unSupportedChains = supportedChains
```

Comment:
unsupported is one word
```suggestion
  const unsupportedChains = supportedChains
```